themes/V3/5ePHB/style.less

```diff
@@ -677,6 +678,136 @@ body {
 		margin-top : 0.2cm;
 	}
 }
+//*****************************
+// *      COVER PAGE
+// *****************************/
+
+.page {
+	&:has(.coverPage):after {
+		display: none;
+	}
+	.coverPage {
```

Comment:
Instead of wrapping the whole coverpage in one big DIV, can we make more use of `:has`? As in, just put `{{coverPage}}` at the top of the page, and that when that class is detected, the whole page will be put into "coverPage" mode.

Reply:
that is a good idea, and then all props can be set as `.page:has(.coverPage) hr` and so on (when processed)

to be hones with you, i don't really see the difference, i would argue that having the entire cover be an element the user understands easier what components make the cover.

Reply:
In short, using `:has` ensures that the *whole page* is the cover. Making it a Div means it's easy for a user to accidently put components outside of the Div.